app/pages/lab/actions/workflow.js

```diff
@@ -0,0 +1,30 @@
+var apiClient = require('panoptes-client/lib/api-client');
+
+var workflows = apiClient.type('workflows');
+
+var workflowActions = {
+  createWorkflow: function(workflowData, sourceWorkflow) {
```

Comment:
The "create" action shouldn't also be responsible for knowing how to copy something. Leave that to the component that accepts a workflow to copy.